### DISCUSSION TITLE
Simple question before I purchase eufy home alarm system

### DISCUSSION THREAD

**Discussion by samturner3** [fuatakgun/eufy_security#244] [Q&A] [answered]
Hi there,

The only issue I have with the eufy home alarm system is no external siren.

Does this integration have an 'in alarm' state that I can use to trigger an external z-wave siren via home assistant? 

Cheers.

**Comment by fuatakgun** [comment, marked as answer]
When system goes into alarm triggered mode, you can do anything home assistant. Yes, it is supported
